boot/freeldr/freeldr/lib/peloader.c

```diff
@@ -352,6 +354,7 @@ WinLdrLoadImage(IN PCHAR FileName,
     Status = ArcSeek(FileId, &Position, SeekAbsolute);
     if (Status != ESUCCESS)
     {
+        WARN("Error while reading %s, Status: %u\n", FileName, Status);
```

Comment:
s/reading/seeking to start of/

Reply:
it's "seeking **the** start of" (and that's too long) / just "seeking" (shorter).